Packs/FeedDHS/ReleaseNotes/2_0_54.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### DHS Feed v2
+
+- Added the extensions field in STIX Domain Object (SDO) indicators. The server now includes custom XSOAR indicator fields as extension properties when serving indicators.
```

Comment:
Added the **Cortex XSOAR Extensions fields** in STIX Domain Object (SDO) indicators. Cortex XSOAR now includes custom XSOAR indicator fields as extension properties when generating indicators.